AUTHORS

```diff
@@ -983,7 +988,99 @@ Yuta Hiroto <git@about-hiroppy.com>
 Ilya Shaisultanov <ishaisultanov@shutterstock.com>
 James Lal <james@silklabs.com>
 Josh Leder <josh@ha.cr>
-surya panikkal <surya.com@gmail.com>
-vsemozhetbyt <vsemozhetbyt@gmail.com>
+Surya Panikkal <surya.com@gmail.com>
+Vse Mozhet Byt <vsemozhetbyt@gmail.com>
```

Comment:
This line should be probably kept as it was, ref: https://github.com/nodejs/node/pull/6373#discussion_r60939956